src/averageAges.js

```diff
@@ -14,12 +14,37 @@
  *
  * @return {number}
  */
+// function genderDetermination(persons, sex) {
+//   return persons.filter(person => person.sex === sex);
+// }
+
+function getAvarageAge(arr) {
+  function callback(prev, newOne) {
+    let sum = prev;
+
+    sum += newOne.died - newOne.born;
+
+    return sum;
+  }
+
+  return arr.reduce(callback, 0) / arr.length;
+}
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // console.log(century)
+
+  const menOnly = people.filter(person => (
+    century !== undefined
```

Comment:
I think you can write just 'century' without '!== undefined' or '===true' and so on. This will already be Boolean value. But I liked your way of solving this task